### PR TITLE
feat: create section about Typescript

### PR DESCRIPTION
**Pull request by mauriciomutte**
I've added section about Typescript

## What was added?
11 Typescript

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1080,3 +1081,74 @@ describe('yourModule', () => {
 **[⬆ back to summary](#-summary)**
 
 ---
+
+## 11. Typescript
+
+- 11.1 [Write tests with "it"](#111-do-not-use-any-type)
```
Comment (lupy100) [inline README.md:1087]:
```suggestion
- 11.1 [Do not use any type](#111-do-not-use-any-type)
```

**Review by lupy100 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mauriciomutte) [reply README.md]: Thanksss!!!

File: README.md [hunk 2]
```diff
@@ -1080,3 +1081,74 @@ describe('yourModule', () => {
...
+## 11. Typescript
+
+- 11.1 [Do not use any type](#111-do-not-use-any-type)
+- 11.2 [Naming convention](#112-naming-convention)
+- 11.3 [Exporting types](#113-exporting-types)
+- 11.4 [Types within a file](#114-types-within-a-file)
+
+### 11.1 Do not use any type
+
+Avoid using `any` type. It is better to use the type that is more specific.
```
Comment (igorwessel) [inline README.md:1094]: **question:** ‏Do you think it's worth mentioning instead  ``avoid any`` prefer use unknown when needs? At least it guarantees that will have to do the typing checks

**Review by felipefialho — APPROVED** [review]

**Review by tassioFront — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (mauriciomutte) [reply README.md]: Very nice suggestion! Added in https://github.com/juntossomosmais/frontend-guideline/pull/28/commits/6753a9d429c1513af1e8ceeefd12ca9f0b6722ea

**Review by igorwessel — APPROVED** [review]

**Pull request merged**